desktop/assets/editorial_features.coffee

```diff
@@ -1,4 +1,6 @@
 require('backbone').$ = $
 $ ->
   if location.pathname is '/2016-year-in-art'
-    require('../apps/editorial_features/components/eoy/client.coffee').init
\ No newline at end of file
+    require('../apps/editorial_features/components/eoy/client.coffee').init
+  else if location.pathname.includes('/venice-biennale')
```

Comment:
Small note—but `includes` is a newer string method that might not be supported by IE or older browsers. Might be good to use `indexOf` or underscore here.